test/controllers/user_test.js

```diff
@@ -137,7 +137,7 @@ describe('User Controller', function() {
 				});
 		});
 
-		it('Cannot update user without valid authentication', function() {
+		it('Cannot update user without valid authentication', function(done) {
```

Comment:
This test was actually broken before you added the `done` to it. Nicely done catching that!